build_all.sh

```diff
@@ -20,12 +20,13 @@ conda install -yq boa
 # load gpuci tools
 source ~/.bashrc
 
-conda mambabuild -c ${CONDA_USERNAME:-rapidsai} -c ${NVIDIA_CONDA_USERNAME:-nvidia} -c conda-forge --python=$PYTHON  \
-    recipes/xgboost
+export RAPIDS_CONDA_BLD_ROOT_DIR='/tmp/conda-bld-workspace'
+export RAPIDS_CONDA_BLD_OUTPUT_DIR='/tmp/conda-bld-output'
 
-conda build -c ${CONDA_USERNAME:-rapidsai} -c ${NVIDIA_CONDA_USERNAME:-nvidia} -c conda-forge --python=$PYTHON  \
-    recipes/xgboost --output > $WORKSPACE/conda-output
+conda mambabuild --python=$PYTHON --croot=$RAPIDS_CONDA_BLD_ROOT_DIR --output-folder=$RAPIDS_CONDA_BLD_OUTPUT_DIR recipes/xgboost
+
+PKGS_TO_UPLOAD=$(find "${RAPIDS_CONDA_BLD_OUTPUT_DIR}" -name "*.tar.bz2")
 
 while read line ; do
     gpuci_retry anaconda -t ${MY_UPLOAD_KEY} upload -u ${CONDA_USERNAME:-rapidsai} --label main --skip-existing $line
-done < $WORKSPACE/conda-output
+done < $PKGS_TO_UPLOAD
```

Comment:
GitHub won't let me make a code suggestion here, but we don't need to use a `while` loop anymore with the `find` method. These lines can be switched to the following:

```sh
gpuci_retry anaconda \
  -t ${MY_UPLOAD_KEY} \
  upload \
  -u ${CONDA_USERNAME:-rapidsai} \
  --label main \
  --skip-existing \
  --no-progress \
  ${PKGS_TO_UPLOAD}
```